### ISSUE TITLE
[UI] Deep link search in /worker-manager

### ISSUE THREAD

**Issue by helfi92** [taskcluster/taskcluster#2124]
**To Reproduce**

1. Go to https://firefox-ci-tc.services.mozilla.com/worker-manager
2. Type "gecko-t/" in the search bar located in the app bar and then press enter.
3. Click on any worker pool id
4. Click back

**Expected behavior**
The list only contains worker pools that start with "gecko-t/".

**Taskcluster version**
23.0.0

**Additional context**
In order to fix this issue, we would need to use a query parameter (e.g., search). An example would be `https://firefox-ci-tc.services.mozilla.com/worker-manager?search=gecko-t%2F`.

**Comment by helfi92** [comment]
Assigned to @Manish-Giri.

**Comment by helfi92** [comment]
@Manish-Giri Are you still interested in finishing this up?

**Comment by gaborszekely** [comment]
@helfi92 Could not replicate this bug.

**Comment by Manish-Giri** [comment]
@helfi92 Yes, I am. Sorry for the delay, I had to step away for a bit, I'll get back on both the issues right away.

**Comment by helfi92** [comment]
> @helfi92 Could not replicate this bug.

I just tried and was able to replicate. What exactly were you not able to replicate?

> @helfi92 Yes, I am. Sorry for the delay, I had to step away for a bit, I'll get back on both the issues right away.

Awesome. Thanks!

**Comment by Manish-Giri** [comment]
@helfi92 Hello! I've been meaning to get your advice on an issue with `/worker-manager`. Whenever I access this page on my local machine, I get this error - 

![Screenshot 2020-06-26 02 02 47](https://user-images.githubusercontent.com/11348778/85826419-8da7b100-b752-11ea-876c-2ee8b63f12ac.png)

I looked around the Network tab, and found this response which causes the error -

![Screenshot 2020-06-26 02 03 38](https://user-images.githubusercontent.com/11348778/85826519-b7f96e80-b752-11ea-918a-8921c3bb5df4.png)

I just wanted to check with you, in case there is some additional setup I should be running to fix this. I already have postgres running in Docker, `web-server` on port 3050, and finally the UI at 5080.

**Comment by helfi92** [comment]
Sorry for the trouble!

@imbstack Should @Manish-Giri use your development environment to continue his UI work or is there another workaround? Thanks.
